docs/core/extensions/options.md

```diff
@@ -27,8 +27,8 @@ Create the following `TransientFaultHandlingOptions` class:
 
 An options class:
 
-* Must be non-abstract with a public parameterless constructor
-* Contain public read-write properties to bind (fields are ***not*** bound)
+- It allows for either parameterless constructors, or for constructors where all parameters have default values
+- Properties can be read-write, or read-init only
```

Comment:
Throughout the C# docs, we're standardizing on the phrase "init only" for these properties:

```suggestion
- Properties can be read-write, or init only
```

Reply:
Your edit removes the requirement that options classes must be non-abstract. Was that intentional?

Reply:
Sort of, I mean they allow records - perhaps it should be a bit more explicit?